server/utils/utils.ts

```diff
@@ -80,9 +80,13 @@ export const retrieveQuestionResponseFromApplication = <T>(
   task: string,
   page: string,
   question?: string,
+  options: { throwIfNotFound?: boolean; returnIfNotFound: unknown } = { throwIfNotFound: true, returnIfNotFound: '' },
```

Comment:
Once it's merged I will use the new function from this commit to do this: https://github.com/ministryofjustice/hmpps-approved-premises-ui/pull/580/commits/a57fd07ac3f3f1eb9ec237439fd204d5cc557792